lib/Demangling/Remangler.cpp

```diff
@@ -2314,6 +2314,10 @@ void Remangler::mangleFullObjCResilientClassStub(Node *node) {
   Buffer << "Mt";
 }
 
+void Remangler::mangleAsyncAnnotation(Node *node) {
```

Comment:
When you have the opportunity, please update the [mangling reference](https://github.com/apple/swift/blob/master/docs/ABI/Mangling.rst)

Reply:
I did... https://github.com/apple/swift/pull/33147/files#diff-3d09d8bf538c5efb52870cdf565b219c